pkg/modifycontroller/controller.go

```diff
@@ -211,6 +215,9 @@ func (ctrl *modifyController) Run(
 	}
 
 	stopCh := ctx.Done()
+
+	go ctrl.slowSet.Run(stopCh)
```

Comment:
Nit: maybe just add a comment saying this is for resource cleanup. Other than this LGTM